docs/install.asciidoc

```diff
@@ -2,7 +2,6 @@
 :awestruct-layout: doc
 :linkattrs:
 :icons: font
-:debezium-version: 0.5.0
 :confluent-platform-version: 3.2.0
```

Comment:
This one should be extracted too and promoted to 3.3.0

Reply:
Done.